src/main/java/io/jomatt/multitenant/sample/web/config/SecurityConfig.java

```diff
@@ -0,0 +1,42 @@
+package io.jomatt.multitenant.sample.web.config;
+
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.security.authentication.AuthenticationManagerResolver;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.web.SecurityFilterChain;
+
+@Configuration
+public class SecurityConfig {
+
+    @Bean
+    @Profile("!prod")
+    public SecurityFilterChain headerFilterChain(HttpSecurity http) throws Exception {
```

Comment:
@jomatt 
After Adding this code in securityConfig 
Swagger URL is working but oauth2 on api call becames 401 

And if i remove this section oauth2 on api is working but swagger url became 401

Scenario
only one application.properties and activated jwt mode
no application-prod.properties

Reply:
@jeenec As described in the `README`, I created two profiles for testing the different scenarios. Both profiles are preconfigured and don't require any changes:
* `default` profile: Header mode
* `prod` profile: JWT mode

If you instead enable JWT mode in the default profile, you should also also adapt the `SecurityConfig` to match your setup, i.e. you need to:
* Remove the `@Profile("prod")` from the `jwtFilterChain` so that this bean is created by default (and not only when the `prod` profile is active)
* Remove the `headerFilterChain` completely or bind it to another `@Profile` so that only the `jwtFilterChain` is used